static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain.js

```diff
@@ -0,0 +1,172 @@
+// @flow
+import $ from 'lib/$';
+import config from 'lib/config';
+import detect from 'lib/detect';
+import template from 'lodash/utilities/template';
+import steadyPage from 'lib/steady-page';
+import getCode from 'commercial/modules/third-party-tags/outbrain-codes';
+import outbrainStr from 'raw-loader!commercial/views/outbrain.html';
+import { loadScript } from 'lib/load-script';
+import checkMediator from 'common/modules/check-mediator';
+import ophan from 'ophan/ng';
+
+const outbrainUrl = '//widgets.outbrain.com/outbrain.js';
+const outbrainTpl = template(outbrainStr);
+
+const selectors = {
+    outbrain: {
+        widget: '.js-outbrain',
+        container: '.js-outbrain-container',
+    },
+    merchandising: {
+        widget: '.js-container--commercial',
+        container: '.js-outbrain-container',
+    },
+    nonCompliant: {
+        widget: '.js-outbrain',
+        container: '.js-outbrain-container',
+    },
+};
+
+const build = function(
+    codes: { code?: string, image?: string, text?: string },
```

Comment:
... and you can reuse it here :-)

You can export types: `export type OutbrainCode`

and import: `import type OutbrainCode from ...`